python/cudf/cudf/core/frame.py

```diff
@@ -35,6 +35,16 @@ class Frame(libcudf.table.Table):
     def _from_table(cls, table):
         return cls(table._data, index=table._index)
 
+    def _mimic_inplace(self, result, inplace=False):
```

Comment:
I made a single `_mimic_inplace` instead of having a separate one for `Series`, `DataFrame`, `Index` and `MultiIndex`.